sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventActionsTableSteps.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.web.application.events;
+
+import static org.sormas.e2etests.pages.application.events.EventDirectoryPage.*;
+
+import cucumber.api.java8.En;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+import javax.inject.Inject;
+import org.apache.commons.lang3.StringUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.web.EventActionTableEntry;
+import org.sormas.e2etests.state.ApiState;
+import org.sormas.e2etests.steps.BaseSteps;
+
+public class EventActionsTableSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  private final BaseSteps baseSteps;
+  private EventActionTableEntry eventActionTableEntry;
+
+  @Inject
+  public EventActionsTableSteps(
+      WebDriverHelpers webDriverHelpers,
+      BaseSteps baseSteps,
+      ApiState apiState,
+      SoftAssertions softly) {
+    this.webDriverHelpers = webDriverHelpers;
+    this.baseSteps = baseSteps;
+
+    Then(
+        "I search last created Event by API using EVENT UUID and wait for (\\d+) entries in the table",
```

Comment:
maybe in the future we shall integrate an up to date cucumber api syntax to avoid regex patterns like (\\d+) and having instead {int} for more readebility.